config/prod/kkdang-genie-rstudio.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: kkdang-genie-compute
```

Comment:
Please update this to match the filename minus the extension

Reply:
? Update what? I was following the same pattern Aaron used for the previous pull request.

Reply:
```suggestion
stack_name: kkdang-genie-rstudio
```